bin/templates/cordova/lib/android_sdk.js

```diff
@@ -52,13 +52,13 @@ module.exports.print_newest_available_sdk_target = function () {
 module.exports.version_string_to_api_level = {
     '4.0': 14,
```

Comment:
Why are not all of these properties changed?

Reply:
Good question, The change was performed eslint automated fixing.

Reply:
I changed it back and disabled eslint around that variable to ensure it remains as strings.